src/encoded/static/components/app.js

```diff
@@ -497,12 +530,48 @@ class App extends React.Component {
         });
     }
 
-    handleAuth0Login(authResult, retrying) {
+    closeSignupModal() {
+        this.setState({ eulaModalVisibility: false });
+    }
+
+    signup(authResult) {
+        const accessToken = authResult.accessToken;
```

Comment:
Are you guaranteed to be able to get accessToken here? authResult is initialized to an empty string.